index.html

```diff
@@ -80,5 +80,15 @@ <h3>Improve how we navigate</h3>
         </div>
       </div>
     </section>
+    <section>
+      <h2>People say</h2>
+      <q class="text-center"
```

Comment:
`blockquote` is better than `q` because this is a _block_ of quote text. `q` is used for inline quotes.

Also, refer to Bootstrap's [`blockquote` docs](https://getbootstrap.com/docs/4.0/content/typography/#blockquotes) in the typography section.